src/aioquic/quic/congestion/cubic.py

```diff
@@ -0,0 +1,204 @@
+from .base import (
+    K_MINIMUM_WINDOW,
+    K_INITIAL_WINDOW,
+    QuicCongestionControl,
+    QuicRttMonitor,
+    register_congestion_control,
+)
+from ..packet_builder import QuicSentPacket
+from typing import Iterable, Dict, Any
+
+# cubic specific variables (see https://www.rfc-editor.org/rfc/rfc9438.html#name-definitions)
+K_CUBIC_K = 1    
+K_CUBIC_C = 0.4
+K_CUBIC_LOSS_REDUCTION_FACTOR = 0.7
+K_CUBIC_MAX_IDLE_TIME = 2   # reset the cwnd after 2 seconds of inactivity
+
+class CubicCongestionControl(QuicCongestionControl):
+    """
+    Cubic congestion control implementation for aioquic
+    """
+
+    def __init__(self, max_datagram_size : int, recovery) -> None:
+        super().__init__(max_datagram_size=max_datagram_size, recovery=recovery)
+        self.additive_increase_factor = max_datagram_size  # increase by one segment
+        
+        self._max_datagram_size = max_datagram_size
+        self._congestion_recovery_start_time = 0.0
+
+        self._rtt_monitor = QuicRttMonitor()
+
+        self.reset()
+
+        self.last_ack = None
+
+        self.now = 0
+
+    def better_cube_root(self, x):
+        if (x < 0):
+            # avoid precision errors that make the cube root returns an imaginary number
+            return -((-x)**(1./3.))
+        else:
+            return (x)**(1./3.)
```

Comment:
Move this to a standalone function, it will avoid re-defining it in `test_cubic.py`